pkg/chains/formats/intotoite6/extract/extract.go

```diff
@@ -62,6 +62,19 @@ func SubjectDigests(obj objects.TektonObject, logger *zap.SugaredLogger) []intot
 		})
 	}
 
+	ssts := artifacts.ExtractStructuredTargetFromResults(obj, artifacts.ArtifactsOutputsResultName, logger)
+	for _, s := range ssts {
+		splits := strings.Split(s.Digest, ":")
+		alg := splits[0]
+		digest := splits[1]
```

Comment:
This can panic if digest isn't what we expect. (we should add tests for this )

Reply:
Yeah we should have separate tests for `SubjectDigests()`, but this particular piece's tested in taskrun/provenance_test.go. I've added a TODO for SubjectDigests() to add more tests in a separate PR

Reply:
The test is present for TaskRun. I added one for PipelineRun